src/library/Uno.Themes.WinUI.Markup/Theme/ComboBox.cs

```diff
@@ -31,6 +31,39 @@ public static class Foreground
 
 						[ResourceKeyDefinition(typeof(Brush), "ComboBoxForegroundFocusedPressed")]
 						public static ResourceValue<Brush> FocusedPressed => new("ComboBoxForegroundFocusedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxUpperPlaceHolderForeground")]
+						public static ResourceValue<Brush> UpperPlaceHolder => new("ComboBoxUpperPlaceHolderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForeground")]
+						public static ResourceValue<Brush> PlaceHolder => new("ComboBoxPlaceHolderForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForegroundPointerOver")]
+						public static ResourceValue<Brush> PlaceHolderPointerOver => new("ComboBoxPlaceHolderForegroundPointerOver", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForegroundPressed")]
+						public static ResourceValue<Brush> PlaceHolderPressed => new("ComboBoxPlaceHolderForegroundPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForegroundDisabled")]
+						public static ResourceValue<Brush> PlaceHolderDisabled => new("ComboBoxPlaceHolderForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForegroundFocused")]
+						public static ResourceValue<Brush> PlaceHolderFocused => new("ComboBoxPlaceHolderForegroundFocused", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxPlaceHolderForegroundFocusedPressed")]
+						public static ResourceValue<Brush> PlaceHolderFocusedPressed => new("ComboBoxPlaceHolderForegroundFocusedPressed", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxLeadingIconForeground")]
+						public static ResourceValue<Brush> LeadingIcon => new("ComboBoxLeadingIconForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxLeadingIconForegroundDisabled")]
+						public static ResourceValue<Brush> LeadingIconDisabled => new("ComboBoxLeadingIconForegroundDisabled", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxArrowForeground")]
+						public static ResourceValue<Brush> Arrow => new("ComboBoxArrowForeground", true);
+
+						[ResourceKeyDefinition(typeof(Brush), "ComboBoxArrowForegroundOpened")]
+						public static ResourceValue<Brush> ArrowOpened => new("ComboBoxArrowForegroundOpened", true);
```

Comment:
These shouldn't be in the main `Foreground` class

Following the specs from https://github.com/unoplatform/uno.themes-private/issues/16#issue-1909235521:

- We should create a `PlaceholderForeground` class with the visual state variants
- We should create a direct property called `UpperPlaceholderForeground` since there are no visual state variants
- We should create a `LeadingIconForeground` nested class with `Default` and `Disabled` as properties
- We should create a `ArrowForeground` nested class with `Default` and `Opened` as properties

Also, `PlaceHolder` should be changed everywhere to `Placeholder` (lowercase h). Update this in the XAML and the docs as well